Utils/src/GoodJetsProducer.cc

```diff
@@ -45,85 +46,87 @@
 
 class GoodJetsProducer : public edm::global::EDFilter<> {
 public:
-   explicit GoodJetsProducer(const edm::ParameterSet&);
-   ~GoodJetsProducer() override;
-   
-   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
-   
-private:
-   bool filter(edm::StreamID, edm::Event&, const edm::EventSetup&) const override;
-   
-   edm::InputTag JetTag_;
-   std::vector<edm::InputTag> SkipTag_;
-   edm::EDGetTokenT<edm::View<pat::Jet>> JetTok_;
-   std::vector<edm::EDGetTokenT<edm::View<reco::Candidate>>> SkipTok_;
-   double maxEta_;
-   double maxNeutralFraction_, maxNeutralFractionHE_, minNeutralFractionHF_, maxPhotonFraction_, minPhotonFractionHE_, maxPhotonFractionHE_, maxPhotonFractionHF_, minChargedFraction_, maxChargedEMFraction_;
-   int minNconstituents_, minNneutralsHE_, minNneutralsHF_, minNcharged_;
-   double jetPtFilter_;
-   bool saveAllId_, saveAllPt_, ExcludeLeptonIsoTrackPhotons_, TagMode_, invertJetPtFilter_;
-   double JetConeSize_;
-
-   
-   // ----------member data ---------------------------
-};
-
-//
-// constants, enums and typedefs
-//
+	explicit GoodJetsProducer(const edm::ParameterSet&);
+	~GoodJetsProducer() override;
 
+	static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
-//
-// static data member definitions
-//
+private:
+	bool filter(edm::StreamID, edm::Event&, const edm::EventSetup&) const override;
+
+	// ----------member data ---------------------------
+	edm::InputTag JetTag_;
+	std::vector<edm::InputTag> SkipTag_;
+	edm::EDGetTokenT<edm::View<pat::Jet>> JetTok_;
+	std::vector<edm::EDGetTokenT<edm::View<reco::Candidate>>> SkipTok_;
+	double maxEta_;
+	//double maxNeutralFraction_, maxNeutralFractionHE_, minNeutralFractionHF_, maxPhotonFraction_, minPhotonFractionHE_, maxPhotonFractionHE_, maxPhotonFractionHF_, minChargedFraction_, maxChargedEMFraction_;
```

Comment:
remove commented-out code (everywhere in this file)

Reply:
Whoops. I meant to do this before. Fixed now.